WordPress/Classes/ViewRelated/Stats/Charts/Charts+Support.swift

```diff
@@ -4,15 +4,15 @@ import Charts
 // MARK: - Charts extensions
 
 extension BarChartData {
-    convenience init(entries: [BarChartDataEntry], valueFormatter: IValueFormatter? = nil) {
-        let dataSet = BarChartDataSet(values: entries, label: nil, valueFormatter: valueFormatter)
+    convenience init(entries: [BarChartDataEntry], valueFormatter: ValueFormatter) {
+        let dataSet = BarChartDataSet(entries: entries, valueFormatter: valueFormatter)
         self.init(dataSets: [dataSet])
     }
 }
 
 extension BarChartDataSet {
-    convenience init(values: [BarChartDataEntry], label: String?, valueFormatter: IValueFormatter?) {
-        self.init(entries: values, label: label)
+    convenience init(entries: [BarChartDataEntry], label: String = "", valueFormatter: ValueFormatter) {
+        self.init(entries: entries, label: label)
```

Comment:
These were the only non-trivial changes, both due by the fact that neither `label` nor `valueFormatter` can be `nil` now.

I decided to use `""` as a replacement for `nil` for `label`. As for `valueFormatter`, turns out we never set a `nil` one so it was straightforward to simply make it not-optional.

Reply:
I looked at this closely and I think `""` as the default for label is the correct choice as well.  Thanks for sharing your thoughts here